### DISCUSSION TITLE
CVE-2023-28849

### DISCUSSION THREAD

**Discussion by iamxhunt3r** [projectdiscovery/nuclei-templates#7215] [General]
Hey, does anyone have any progress on CVE-2023-28849 ? 
Seems like a critical bug out there.

https://github.com/glpi-project/glpi/security/advisories/GHSA-9r84-jpg3-h4m6

**Comment by princechaddha** [comment]
Hello, @iamxhunt3r! At present, our team isn't working on CVE-2023-28849. However, please don't hesitate to create an issue with relevant references or POC, so that we can write a template for it.
